monkey/monkey_island/cc/app.py

```diff
@@ -109,7 +110,15 @@ class CustomConfirmRegisterForm(ConfirmRegisterForm):
             "Email", default="dummy@dummy.com", validators=[validate_no_user_exists_already]
         )
 
-    app.security = Security(app, user_datastore, confirm_register_form=CustomConfirmRegisterForm)
+    login_manager = LoginManager()
+    login_manager.init_app(app)
```

Comment:
Why?

Reply:
Do we not need this if we want to use flask-login? Or is that done by the LgoinForm?

Reply:
We don't want to use `flask-login`, we want to use `flask-security-too`